### ISSUE TITLE
Error on parsing JSX with hard spaces in a new line and end of line

### ISSUE THREAD

**Issue by ukabuer** [Rich-Harris/buble#46]
For example

``` jsx
<div>
  <a>1</a>&nbsp;
  &nbsp;
</div>
```

is compiled to

``` js
React.createElement( 'div', null,
  React.createElement( 'a', null, "1" )&nbsp;
  &nbsp;
)
```

here `&nbsp;` is an unexpected token.

BTW I think it would be nice to update the report bug link at https://buble.surge.sh to this repo

**Comment by adrianheine** [comment]
If done the latter in Rich-Harris/buble-demo#3.

**Comment by Rich-Harris** [comment]
fixed in 0.18
